### PR TITLE
Fixes #23278: Improve some messages in the interface - plugins

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/23278

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
PR updated with a new commit

**Review by fanf — COMMENTED** [review]
Lgtm (and wow!)

File: change-validation/src/main/resources/template/ChangeValidationManagement.html [hunk 1]
```diff
@@ -242,13 +274,12 @@ <h3 class="page-subtitle">Configure Groups with Change Validations</h3>
         </div>
       </div>
     </div>
-  </div>
```
Comment (ElaadF) [inline change-validation/src/main/resources/template/ChangeValidationManagement.html:245]: Remove this one doesn't seem right, but I am not sure

**Comment by amousset** [comment]
PR updated with a new commit

**Review by ElaadF — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**